openstack/lyra/templates/runs-pruning.yaml

```diff
@@ -0,0 +1,31 @@
+kind: Deployment
+apiVersion: extensions/v1beta1
+
+metadata:
+  name: runs-pruning
+
+spec:
+  revisionHistoryLimit: 5
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: runs-pruning
+    spec:
+      containers:
+        - name: pruning
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{.Values.image.pullPolicy}}
+          args:
+            - sh
+            - -ec
+            - |
+              while true; do
+                if [ $(date '+%H%M') = '{{ .Values.session_cleanup_time }}' ]; then
+                  bin/rake run:cleanup
+                  echo "bin/rake run:cleanup exited"
+                fi
+                sleep 55
+              done
+          env:
+{{ include (print .Template.BasePath "/_env.tpl") . | indent 12 }}
```

Comment:
Has to be `{{ include "env.vars" . | indent 12 }}`. See api-deployment.yaml

Reply:
right! i was missing something I couldn't remember...